content/browser/renderer_host/frame_connector_delegate.h

```diff
@@ -181,6 +181,13 @@ class CONTENT_EXPORT FrameConnectorDelegate {
   // creating a window in the ui server so that this can render to the screen.
   virtual void EmbedRendererWindowTreeClientInParent(
       ui::mojom::WindowTreeClientPtr window_tree_client) {}
+
+  // Tells the delegate FrameSinkId has been changed. It's extremely important
```

Comment:
nit: I would remove "extremely" terminology here.

Maybe something like:

````
// Tells the delegate FrameSinkId has been changed
// When mus is hosting viz, the delegate must react to FrameSinkId changes, in order to avoid out of sync between XXX and YYY. RenderWidgetHostViewChildFrame calls this method in order to create ...
````

Reply:
"// Tells the delegate FrameSinkId has been changed" is actually enough here